draft-ietf-lake-edhoc.md

```diff
@@ -752,7 +775,7 @@ message_1 = (
   METHOD : int,
   SUITES_I : suites,
   G_X : bstr,
-  C_I : bstr / int,
+  C_I : bstr,
```

Comment:
Also consistently with the later editing in the "CDDL Definitions" appendix, this should be

`C_I : bstr / -24..23,`

Reply:
Overtaken by later proposals